styles/transition_effect.h

```diff
@@ -14,41 +16,7 @@
 // EFFECT_COLOR using TRANSITION1. Then transition back using TRANSITION2.
 
 template<class TRANSITION, BladeEffectType EFFECT>
-class TransitionEffectL {
-public:
-  LayerRunResult run(BladeBase* blade) {
-    if (effect_.DetectScoped(blade)) {
-      transition_.begin();
-      run_ = true;
-    }
-    if (run_) {
-      transition_.run(blade);
-      if (transition_.done()) run_ = false;
-    }
-    last_detected_blade_effect = nullptr;
-    if (!run_) {
-      // All effects now wake the blade up again if needed, so it's safe to
-      // always return TRANSPARENT_UNTIL_IGNITION
-      return LayerRunResult::TRANSPARENT_UNTIL_IGNITION;
```

Comment:
TransitionPulseL will need some optimizations before we can do this, otherwise blades that use TransitionEffectL will never turn off the power, resulting in poor standby time.